front/components/sparkle/AppLayout.tsx

```diff
@@ -214,6 +282,7 @@ export default function AppLayout({
   children: React.ReactNode;
 }) {
   const { sidebarOpen, setSidebarOpen } = useContext(SidebarContext);
+  const { laptopNavOpen, setLaptopNavOpen } = useContext(NavigationBarContext);
```

Comment:
laptop?

Reply:
That basically is the sidebar for non mobile devices, right? But agree naming can be better will change that 😄